vm/vm.go

```diff
@@ -433,6 +433,34 @@ func (vm *VM) Initialize(
 	webSocketServer, pubsubServer := rpc.NewWebSocketServer(vm, vm.config.StreamingBacklogSize)
 	vm.webSocketServer = webSocketServer
 	vm.handlers[rpc.WebSocketEndpoint] = pubsubServer
+
+	has, err = vm.HasLastProcessed()
+	if err != nil {
+		snowCtx.Log.Error("could not determine if have last processed")
+		return err
+	}
+	if has {
```

Comment:
Once this is moved to a separate function, we can also add an early return here since there's nothing else happening after it.

Then we can make this:

```golang
if !has {
    return nil
}
```

and then the remainder can go outside of the if block, which is a bit cleaner

Reply:
It's cleaner! Thanks!